package exception;

public class Try {
	
	public static void main(String[] args)
	
	{
		try{
			
			System.out.println("Try Block");
		}
		
		catch()
		{
			System.out.println("Catch block");
		}
		
	}

}
